File: EPS BackEnd/EPS.WebServices/EPS.Common.Models/Material/MaterialRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EPS.Common.Models
{
    public class MaterialRequest
    {
        public string UserCode { get; set; }
        /// <summary>
        /// Historia
        /// </summary>
        public int History { get; set; }
        /// <summary>
        /// Medicamento
        /// </summary>
        public Material Material { get; set; }
    }
}

File: EPS FrontEnd/EPS/src/app/models/Material/material-request.ts
import { Material } from './material';

///Parametros peticion material
export class MaterialRequest
{
  UserCode: string;
  /// <summary>
  /// Historia
  /// </summary>
  History: number;
  /// <summary>
  /// Medicamento
  /// </summary>
  Material: Material;
}
